src/re_unicode.rs

```diff
@@ -1198,7 +1199,7 @@ pub trait Replacer {
     /// be beneficial to avoid finding sub-captures.
     ///
     /// In general, this is called once for every call to `replacen`.
-    fn no_expansion<'r>(&'r mut self) -> Option<Cow<'r, str>> {
+    fn no_expansion(&mut self) -> Option<Cow<str>> {
```

Comment:
As with above, please put the lifetime back. Leaving it off the implementations is fine.